src/otx/algo/classification/backbones/mobilenet_v3.py

```diff
@@ -376,14 +375,10 @@ def __init__(self, mode: str = "large", width_mult: float = 1.0, **kwargs):
             self.key = self.key + f"_{int(width_mult * 100):03d}"  # pylint: disable=consider-using-f-string
         self.init_weights(self.pretrained)
 
-    def forward(self, x):
```

Comment:
is it ok to remove this function?

Reply:
It was duplicate code between the four Class structures.